setup/setup.sh

```diff
@@ -11,6 +11,7 @@ for var in $vars; do
 done
 
 echo "::group::Set up artifactory"
+ARTIFACTORY_USERNAME=${ARTIFACTORY_USERNAME_INPUT:-ARTIFACTORY_USERNAME}
```

Comment:
If no username was given through the input, this would always overwrite the existing value in `ARTIFACTORY_USERNAME` with the specified string, not the existing value, essentially breaking authentication.

Reply:
Fixed